lib/ui/screens/group_screen.dart

```diff
@@ -0,0 +1,201 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_widget_from_html_core/flutter_widget_from_html_core.dart';
+import 'package:reaxit/api/api_repository.dart';
+import 'package:reaxit/blocs/group_cubit.dart';
+import 'package:reaxit/models/group.dart';
+import 'package:reaxit/models/member.dart';
+import 'package:reaxit/ui/widgets/cached_image.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+import '../widgets/app_bar.dart';
+import '../widgets/error_scroll_view.dart';
+import '../widgets/member_tile.dart';
+
+class GroupScreen extends StatefulWidget {
+  final int pk;
+  final Group? group;
+
+  GroupScreen({required this.pk, this.group}) : super(key: ValueKey(pk));
+
+  @override
+  State createState() => _GroupScreenState();
+}
+
+class _GroupScreenState extends State<GroupScreen> {
+  late ScrollController _controller;
+
+  late final GroupCubit _groupCubit;
+
+  @override
+  void initState() {
+    final api = RepositoryProvider.of<ApiRepository>(context);
+    _groupCubit = GroupCubit(api, pk: widget.pk)..load();
+    _controller = ScrollController(); //..addListener(_scrollListener());
+    super.initState();
+  }
+
+  @override
+  void dispose() {
+    _groupCubit.close();
+    super.dispose();
+  }
+
+  Widget _makeImage(ListGroup group) {
+    return Stack(
+      fit: StackFit.loose,
+      children: [
+        CachedImage(
+            imageUrl: group.photo.full,
+            placeholder: 'assets/img/default-avatar.jpg')
+      ],
+    );
+  }
+
+  Widget _makeDescription(ListGroup group) {
+    return Padding(
+        padding: const EdgeInsets.symmetric(horizontal: 0, vertical: 4),
+        child: HtmlWidget(
+          group.description,
+          onTapUrl: (String url) async {
+            try {
+              await launch(
```

Comment:
Perhaps this should (same goes for the html description on events) check for deep links: 
```dart
if (isDeepLink(uri)) {
  context.go(Uri(
    path: uri.path,
    query: uri.query,
  ).toString());
} else {
```